content/bn/virtual-machine.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ভার্চুয়াল মেশিন
```

Comment:
You should have `ভার্চুয়াল মেশিন (Virtual Machine)`  instead of the Localized content only